libsrc/flatbufserver/FlatBufferServer.cpp

```diff
@@ -93,14 +99,27 @@ void FlatBufferServer::startServer()
 {
 	if(!_server->isListening())
 	{
-	    if(!_server->listen(QHostAddress::Any, _port))
-	    {
-	        Error(_log,"Failed to bind port %d", _port);
-	    }
-	    else
-	    {
-	        Info(_log,"Started on port %d", _port);
-	    }
+		if(!_server->listen(QHostAddress::Any, _port))
+		{
+			Error(_log,"Failed to bind port %d", _port);
+		}
+		else
+		{
+			Info(_log,"Started on port %d", _port);
+#ifdef ENABLE_AVAHI
+			if(_serviceRegister == nullptr)
```

Comment:
@Paulchen-Panther This is documented but not implemented. I am not missing anything right ?

Reply:
@brindosch made the last change to the AVAHI interface. Maybe you should check his PR to see if he has deactivated something that is missing now.
https://github.com/hyperion-project/hyperion.ng/pull/738
I can't say much without looking deeper because @brindosch introduced the AVAHI interface. Sorry

Reply:
Avahi support seems to be implemented before that PR and I don't see FlatBuffer advertisement there as well. Probably forgotten. 
Thanks.